### PR TITLE
Final conversion of api-experimental to web action (not experimental)

### PR DESCRIPTION
**Pull request by mdeuser**
Actions changes:
- Change parameters used (i.e. no longer use __ow_meta_*)
- Resolve()/Reject() responses now in .http format

CLI changes:
- Invoke api gw actions at .http for v1/experimental commands
- Better handling/messaging of error responses

### CODE REVIEW

**Comment by mdeuser** [comment]
PG3/624

@csantanapr - ready for review/comments/merge

**Comment by mdeuser** [comment]
@dubeejw - would you please take a look at the cli/go changes.  thanks!

File: tools/cli/go-whisk/whisk/client.go [hunk 1]
```diff
@@ -375,14 +375,11 @@ func parseApplicationError(resp *http.Response, data []byte, v interface{}) (*ht
...
-    // Handle application errors that occur with blocking invocations when result is true (#5)
+    // Handle application errors that occur with blocking invocations when --result option is true (#5)
     if err == nil && appErrResult.Error != nil {
         Debug(DbgInfo, "Error code is null, blocking with result invocation error has occured\n")
-        errMsg := wski18n.T(
-            "The following application error was received: {{.err}}",
-            map[string]interface{}{
-                "err": *appErrResult.Error,
-            })
+        errMsg := fmt.Sprintf("%v", *appErrResult.Error)
```
Comment (dubeejw) [inline tools/cli/go-whisk/whisk/client.go]: Does this return the application error from a normal action??

Comment (mdeuser) [inline tools/cli/go-whisk/whisk/client.go]: yep (when blocking/result options were set on the invocation).  the only change was to remove the `The following application error was received` preamble which cluttered up the output.

**Comment by mdeuser** [comment]
pg2/1614

@csantanapr - ready for merging

**Pull request merged**